### ISSUE TITLE
Close drawer after submitting code

### ISSUE THREAD

**Issue by smcmurtry** [cds-snc/covid-shield-mobile#76]
This should be closed automatically:
<img width="556" alt="Screen Shot 2020-06-19 at 3 36 15 PM" src="https://user-images.githubusercontent.com/5498428/85181283-bc99c080-b242-11ea-874b-fd75f5d242b4.png">

**Comment by Stephen-PM** [comment]
The menu should retract after completing an action, like entering a key or turning on notifications.

**Comment by timarney** [comment]
https://github.com/CovidShield/mobile/issues/175
